internal/corazawaf/transaction.go

```diff
@@ -1839,66 +1833,239 @@ func (v *TransactionVariables) ArgsPostNames() collection.Collection {
 	return v.argsPostNames
 }
 
+func (v *TransactionVariables) All(f func(v variables.RuleVariable, col collection.Collection) bool) {
```

Comment:
Please do add documentation to this function as it isn't straightforward why we iterate over the two of them (I feel like it is convenience) but no 100% sure.

Reply:
Also maybe add a TODO to generate this code in the future?

Reply:
While generating code is something to consider it's not obviously good or bad here I think so will leave out the TODO